src/manifest.json

```diff
@@ -14,12 +14,13 @@
     {
       "matches": ["*://github.com/*"],
       "js": ["contentScript.bundle.js"],
-      "run_at": "document_end"
+      "css": ["contentScript.css"],
+      "run_at": "document_start"
```

Comment:
`run_at` has been changed from `document_end` to `document_start`, which makes the content script run as early as possible so Hypercrx does not need to wait for all GitHub elements to be parsed to inject its charts, resulting in better user experience.

The timing of `document_start` makes `awaitDomReady` useful. if we kept `run_at` as `document_end`, we would not need `awaitDomReady` because the content script would always run after the DOM was ready.